proposals/lip-0052.md

```diff
@@ -1394,7 +1391,7 @@ def getCollectionID(nftID: NFTID) -> CollectionID:
 This function returns the `attributesArray` of an NFT.
 
 ```python
-def getAttributesArray(nftID: NFTID) -> List[{Module, Attributes}]:
+def getAttributesArray(nftID: NFTID) -> List[{Module, bytes}]:
```

Comment:
```suggestion
def getAttributesArray(nftID: NFTID) -> List[dict[Module, bytes]]:
```

Reply:
If you use capital `List` you have to import it frm typing, while from python 3.5 I think u can just use `list`